src/coreclr/jit/lowerxarch.cpp

```diff
@@ -4006,6 +4006,36 @@ GenTree* Lowering::TryLowerAndOpToAndNot(GenTreeOp* andNode)
     return andnNode;
 }
 
+void Lowering::LowerBswapOp(GenTreeOp* node)
+{
+    assert(node->OperIs(GT_BSWAP));
+
+    if (!comp->compOpportunisticallyDependsOn(InstructionSet_MOVBE))
+    {
+        return;
+    }
+
+    GenTree* operand = node->gtGetOp1();
+    if (operand->OperIs(GT_IND))
+    {
+        operand->SetContained();
+
+        return;
+    }
+
+    LIR::Use use;
+    if (!BlockRange().TryGetUse(node, &use))
+    {
+        return;
+    }
+
+    GenTree* user = use.User();
+    if (user->OperIs(GT_STOREIND))
+    {
+        node->SetContained();
```

Comment:
Same comment as above -- there can be interference between the `GT_BSWAP` and `GT_STOREIND` nodes.

Reply:
Spoke with @SingleAccretion about this one on Discord. Because the BSWAP node itself is side effect-less its redundant to check for the interference. I have omitted the check for now.